### ISSUE TITLE
Deprecate Subreddit.Best

### ISSUE THREAD

**Issue by sirkris** [sirkris/Reddit.NET#125]
I should never have put that in the Subreddit controller to begin with.  My bad.  With the FrontPage property now available after #121, combined with the fact that it's a global sort that doesn't work with subreddits, this one seems like a no-brainer.

For BC compatibility, I'll leave it there but add a deprecation flag, instead.

**Comment by sirkris** [comment]
Merged into 1.5.
